### PR TITLE
pedestrian_tracker: generalize detection

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: demos/pedestrian_tracker_demo/src/detector.cpp [hunk 1]
```diff
@@ -99,20 +106,37 @@ ObjectDetector::ObjectDetector(
...
+    switch(inputInfo.size()) {
+        case 1: {
+            InputInfo::Ptr inputInfoFirst = inputInfo.begin()->second;
+            inputInfoFirst->setPrecision(Precision::U8);
+            inputInfoFirst->getInputData()->setLayout(Layout::NCHW);
+            input_name_ = inputInfo.begin()->first;
+            im_info_name_.clear();
+            break;
+        }
+        case 2: {
```
Comment (IRDonch) [inline demos/pedestrian_tracker_demo/src/detector.cpp]: Doesn't the `case 2` code also work for `case 1`?

**Comment by IRDonch** [comment]
This is to support `person-detection-retail-0002`, right? If so, please add a corresponding case to the test script.

**Comment by IRDonch** [comment]
(By the way, the tests failed because the build ran on the wrong node. I fixed the pipeline to make sure that doesn't happen anymore.)

File: demos/pedestrian_tracker_demo/src/detector.cpp (hunk 1 above)
Comment (Wovchena) [reply demos/pedestrian_tracker_demo/src/detector.cpp]: Updated

**Comment by Wovchena** [comment]
> is is to support `person-detection-retail-0002`, right? If so, please add a corresponding case to the test script.

Added

**Pull request merged**